lib/java/gradle/sourceConfiguration.gradle

```diff
@@ -31,7 +31,7 @@
 // also a runtime CI that's based on Java 11 to ensure that.
 java {
     toolchain {
-        languageVersion = JavaLanguageVersion.of(17)
+        languageVersion = JavaLanguageVersion.of(19)
```

Comment:
so with this definition - java defined with github actions setup is onoy used for Gradle process and not to build/run tests?

Reply:
I think this forces developers to build using JDK 19. I would prefer not to do that. I think many developers are perfectly content to build using the latest LTS, which I think is still 17. The build should still work with later versions, but I don't think the build should require anything later than 17. I'm not very experienced with Gradle, but I think that's what this does.

Can we bump to the newest Gradle without forcing devs to develop using newer than JDK 17?